server/api.js

```diff
@@ -27,4 +27,29 @@ router.post("/videos", async (req, res) => {
 	res.status(200).json({ success: true, data: { id: newVideoId } });
 });
 
+router.delete("/videos/:id", async (req, res) => {
+	const videoId = req.params.id;
+
+	try {
+		const checkQuery = await db.query("SELECT * FROM videos WHERE id = $1", [
```

Comment:
It's  good to see you preventing SQL Injection by using parameters! 🙌 If you’re not sure what I'm  on about, Google SQL Injection. @RbAvci This is a good example of preventing SQL Injection.